Написаны тесты на сценарии задания.

Vezaris opened the pull request:
Написаны тесты на сценарии задания.
Добавлено описание в README.

Vezaris committed:
Написаны тесты на сценарии задания.
Добавлено описание в README.

pomegranate282922 commented on src/main/java/ru/practicum/yandex/Model.java:
```diff
@@ -0,0 +1,142 @@
+
+public class Model {
+                                    /** Эндпоинты **/
+    final String REGISTER_USER = "auth/register";
```
Везде добавить статик чтобы это было классовой константой 
Или переименовать в camelCase 
Нужно исправить

pomegranate282922 commented on src/main/java/ru/practicum/yandex/Model.java:
```diff
@@ -0,0 +1,142 @@
+    }
+
+    @Step("Отправить запрос на авторизацию пользователя")
+    public Response sendPostAuth(Object object) {
```
В параметрах надо передавать не Object , а созданный класс pojo, соответствующий данному json 
Нужно исправить везде

pomegranate282922 commented on src/test/java/ru/practicum/yandex/OrderCreateTest.java:
```diff
@@ -0,0 +1,93 @@
+
+    @Before
+    public void setUp() {
+        RestAssured.baseURI = "https://stellarburgers.nomoreparties.site/api/";
```
Base uri устанавливаем один раз в проекте 
Нужно исправить

pomegranate282922 commented on src/test/java/ru/practicum/yandex/UserDataEditTest.java:
```diff
@@ -0,0 +1,82 @@
+    @Before
+    public void setUp() {
+        RestAssured.baseURI = "https://stellarburgers.nomoreparties.site/api/";
+        user = new User("TestMail@yandex.ru", "testPassword1", "testName1");
```
Лучше использовать рандомизацию 
Можно улучшить

pomegranate282922 commented on src/test/java/ru/practicum/yandex/UserRegistryTest.java:
```diff
@@ -0,0 +1,78 @@
+
+    @After
+    public void deleteUser() {
+        model.deleteUser(user);
```
Отлично что везде есть удаление

pomegranate282922 reviewed (commented):
Работа хорошая! Все тесты реализованы. Вызовы ручек вынесены из тестов. Везде есть удаление тестовых данных 
Замечания:
1. Нейминг 
2. Base uri устанавливаем один раз и выносим урл в константы 
3. Вместо Object используем конкретные классы 
4. Остальное по мелочи

Vezaris committed:
Исправлены замечания, а именно:
- добавлен static для классовых констант
- Object заменен на конкретный класс
- Base uri теперь переиспользуется

pomegranate282922 commented on src/test/java/ru/practicum/yandex/UserDataEditTest.java:
```diff
@@ -0,0 +1,72 @@
+
+  @Before
+  public void setUpForUserDataEdit() {
+    user2 = new User();
```
Не используем магические цифры в переменных. 

Нужно исправить

pomegranate282922 reviewed (approved):
Нейминг в одном месте можно улучшить, но уверена, что ты справишься самостоятельно.
Зачет!
